go.mod

```diff
@@ -95,6 +95,8 @@ require (
 	sourcegraph.com/sourcegraph/appdash-data v0.0.0-20151005221446-73f23eafcf67
 )
 
+require golang.org/x/exp v0.0.0-20220426173459-3bcf042a4bf5
```

Comment:
Merge to above block?

Reply:
It is from the ```util/math```. I restart to format it and put it into the front part together.